### PR TITLE
Fix issues on some catalog pages

### LINKED ISSUE #34889: Bugs on migrated features and feature values page
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

There are many issues on the migrated features page.
- Extra message that features are enabled on store. It wasn't there on the legacy page and just wastes space.
- Feature grid rows are not clickable.
- Feature value grid rows are not clickable.
- When adding new feature value and clicking cancel, you are always redirected to feature listing, instead of value listing.

### Expected behavior

_No response_

### Steps to reproduce

1. Get develop branch.
2. Check Catalog > Attributes and features

### PrestaShop version(s) where the bug happened

develop

### PHP version(s) where the bug happened

_No response_

### If your bug is related to a module, specify its name and its version

_No response_

### Your company or customer's name goes here (if applicable).

_No response_

### PR DESCRIPTION
**Pull request by Hlavtox**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | See below
| Type?             | bug fix
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | Tests green + verify the list.
| UI Tests          | https://github.com/florine2623/testing_pr/actions/runs/7785727983 ✅ 
| Fixed issue or discussion?     | Fixes #34889
| Related PRs       | 
| Sponsor company   | 

### List of changes and things fixed
- Feature value form cancel link will now lead you to the page you came from, not to the listing.
- When saving feature value, it will now redirect you to the page you came from, not to the listing.
- Feature list is now clickable. Like on legacy page.
- Feature value list is now clickable. Like on legacy page.
- Removed unused horizontal controller.
- Removed message "Features are enabled" that displayed all the time and just wasted space.
Snímek obrazovky 2023-12-21 171738
- Removed message "Brands are enabled" that displayed all the time and just wasted space.
Snímek obrazovky 2023-12-21 171720
- Removed message "Suppliers are enabled" that displayed all the time and just wasted space.
Snímek obrazovky 2023-12-21 171728
- Added message `No products has been found for this brand.` when viewing a brand with no products. Ignore the big list of changes in the twig, it's just because it got tabbed.
Snímek obrazovky 2023-12-21 171826

### CODE REVIEW

**Review by matthieu-rolland — COMMENTED** [review]
@MatShir do you think you need to validate the issue (and proposed improvements ?)

**Comment by Hlavtox** [comment]
@matthieu-rolland The PR just restores the behavior of previous PS versions

**Comment by Hlavtox** [comment]
@PrestaShop/product-council Can someone confirm? :-)

**Review by ShaiMagal — DISMISSED** [review]

**Comment by MatShir** [comment]
LGTM ✅

**Review by boherm — DISMISSED** [review]

**Comment by florine2623** [comment]
Hello @Hlavtox ,

Tests are red, could you check please ? ^^

**Comment by Hlavtox** [comment]
@PrestaShop/ui-tests-maintainers Hi guys, could you please adapt the tests on my branch and remove the _it(`should check that the message alert contains '${test.args.action}'`, async function () {_ tests? 

Files: `09_enableDisableBrands.ts` + `08_enableDisableSuppliers.ts`

**Review by M0rgan01 — DISMISSED** [review]

**Comment by nesrineabdmouleh** [comment]
UI tests => https://github.com/nesrineabdmouleh/testing_pr/actions/runs/7693944410

**Review by nesrineabdmouleh — APPROVED** [review]

**Review by kpodemski — APPROVED** [review]
Great 👍🏻

**Review by Progi1984 — APPROVED** [review]
Ok for UI Tests

**Comment by Hlavtox** [comment]
Thak you @Progi1984 & @nesrineabdmouleh!!! :-)

@florine2623 New test run here https://github.com/Hlavtox/ga.tests.ui.pr/actions/runs/7725557101

File: src/PrestaShopBundle/Resources/config/routing/admin/sell/catalog/features.yml [hunk 1]
```diff
@@ -6,13 +6,6 @@ admin_features_index:
     _legacy_controller: AdminFeatures
     _legacy_link: AdminFeatures
 
-admin_features_horizontal_index:
```
Comment (jolelievre) [inline src/PrestaShopBundle/Resources/config/routing/admin/sell/catalog/features.yml]: Hmm it's weird, this is supposed to be cleaned already in this PR https://github.com/PrestaShop/PrestaShop/pull/35191 🤔 Maybe your branch needs to be rebased?

Comment (Hlavtox) [reply src/PrestaShopBundle/Resources/config/routing/admin/sell/catalog/features.yml]: Yeah, but I think that as it's a resolvable change and the same code, git is not complaining. :-) It's an older PR.

**Review by boherm — APPROVED** [review]

**Comment by florine2623** [comment]
@Hlavtox the tests are still red 😕  could you check ?

**Comment by nesrineabdmouleh** [comment]
@florine2623 
UI tests https://github.com/nesrineabdmouleh/testing_pr/actions/runs/7742115066 :green_circle:

**Comment by Hlavtox** [comment]
Thank you @nesrineabdmouleh! 🌷I tried to reload the tests, no idea why it didn't work. 🤷‍♂️

**Comment by nesrineabdmouleh** [comment]
> Thank you @nesrineabdmouleh! 🌷I tried to reload the tests, no idea why it didn't work. 🤷‍♂️

You're welcome :)
You need to synchronise your fork (Update last week) ;)

**Comment by Hlavtox** [comment]
Aaaaahaaa, OK, I will remember next time!

**Review by florine2623 — COMMENTED** [review]
Hello @Hlavtox ,

Checked the following :

- Feature value form cancel link will now lead you to the page you came from, not to the listing. ✅ 
- When saving feature value, it will now redirect you to the page you came from, not to the listing. ✅ 
- Feature list is now clickable. Like on legacy page. ✅  Double-click to open a feature with feature value / Single-click to open a feature without value
- Feature value list is now clickable. Like on legacy page. ✅ 
- Removed unused horizontal controller. ✅ I just checked that no regression were found
- Removed message "Features are enabled" that displayed all the time and just wasted space. ✅ 
- Removed message "Brands are enabled" that displayed all the time and just wasted space. ✅ 
- Removed message "Suppliers are enabled" that displayed all the time and just wasted space. :x: i still see it, could you check ? :D
- Added message No products has been found for this brand. when viewing a brand with no products. Ignore the big list of changes in the twig, it's just because it got tabbed. ✅ 


Tiny tiny thing ^^could you check :D

**Comment by Hlavtox** [comment]
@florine2623 Florine could you try to reproduce again? I just rebased it to be sure, but it works correctly on my end...
Suppliers enabled - no message.
Suppliers disabled - message.

**Review by florine2623 — COMMENTED** [review]
Hello again @Hlavtox ,

Installed again the PR and tested all the fix ! it looks good to me ✅ 

Relaunched the auto tests : https://github.com/florine2623/testing_pr/actions/runs/7811958730
They're broken :x: 
@nesrineabdmouleh could you check ?

**Comment by Hlavtox** [comment]
Ping @florine2623 @nesrineabdmouleh Florine could you try to restart the one test?

**Comment by florine2623** [comment]
Thanks ! Tests are all green ! 
https://github.com/florine2623/testing_pr/actions/runs/7811958730

**Pull request merged**